demo-saas/api/src/content-scope/content-scope.interface.ts

```diff
@@ -1,8 +0,0 @@
-// eslint-disable-next-line @typescript-eslint/no-unused-vars, unused-imports/no-unused-imports
-import { ContentScope } from "@comet/cms-api";
-import { type ContentScope as BaseContentScope } from "@src/site-configs";
-
-declare module "@comet/cms-api" {
-    // eslint-disable-next-line @typescript-eslint/no-empty-object-type
-    interface ContentScope extends BaseContentScope {}
-}
```

Comment:
I guess we should keep the module augmentation for the content scope.

Reply:
added it here again:
4ac0efa057f1e0be7b386ec9535bc566e9f239f0